StructureDefinition/NHSDigital-Specimen-Genomics.StructureDefinition.json

```diff
@@ -142,32 +148,13 @@
         "min": 1
       },
       {
-        "id": "Specimen.collection.extension",
+        "id": "Specimen.collection.extension:specialHandling",
         "path": "Specimen.collection.extension",
-        "slicing": {
-          "discriminator": [
-            {
-              "type": "value",
-              "path": "url"
-            }
-          ],
-          "rules": "open"
-        },
-        "min": 0
-      },
-      {
-        "id": "Specimen.collection.extension:specimen-specialHandling",
-        "path": "Specimen.collection.extension",
-        "sliceName": "specimen-specialHandling",
-        "min": 0,
-        "type": [
-          {
-            "code": "Extension",
-            "profile": [
-              "http://hl7.org/fhir/StructureDefinition/specimen-specialHandling"
-            ]
-          }
-        ]
+        "sliceName": "specialHandling",
+        "binding": {
+          "strength": "extensible",
+          "valueSet": "https://fhir.nhs.uk/ValueSet/genomics-specimen-collection-specialhandling"
```

Comment:
Given the special handling extension does not relate to conditions (now related to disposal of the specimen), could you remove the gen-1 constraint on the profile.
This binding strength is fine for now, but we may also want to consider reducing the binding strength if other codes end up being used.